src/test/scala/io/iohk/ethereum/db/storage/ReferenceCountNodeStorageSpec.scala

```diff
@@ -28,20 +28,20 @@ class ReferenceCountNodeStorageSpec extends FlatSpec with Matchers {
   }
 
   it should "not remove a key that was inserted after deletion when pruning" in new TestSetup {
-    val storage = new ReferenceCountNodeStorage(nodeStorage, blockNumber = Some(1))
+    val storage = new ReferenceCountNodeStorage(nodeStorage, bn = 1)
```

Comment:
maybe it could be some `mkNodeStorage` method in `TestSetup` that accept only block number, because nodeStorage is unchanged

Reply:
we would also need to pass `CachedNodeStorage` of `NodeStorage` so i am not sure if we would gain anything by it.